src/main/java/homework_7/Main.java

```diff
@@ -0,0 +1,9 @@
+package homework_7;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Code conventions are generally not followed, and Functional interface is created with mistakes (you declare generics, but don't use them)